src/character/character-sheet/edit/EditResources.tsx

```diff
@@ -17,8 +17,56 @@ import { useParams } from "react-router-dom";
 import { Theme } from "../../../Theme";
 import { Character } from "../../../utils/types/Character";
 import { useObject } from "react-firebase-hooks/database";
-import { snapshotToValue } from "../../../utils/firebase/SnapshotFormatter";
+import {
+  dataObjectToList,
+  snapshotToValue,
+} from "../../../utils/firebase/DataFormatting";
 import { Button } from "../../../shared/buttons/Button";
+import { AddBuffModal } from "./AddBuffModal";
+import {
+  addTemporaryEffect,
+  tickBuffDurationDown as tickTemporaryEffectDuration,
+} from "../../../services/temporary-effects-service";
+import { FlexRow } from "../../../codex/shared/FlexRow";
+import { TextElement } from "../../../shared/text/Text";
+
+const PageContens = styled.div({
+  display: "flex",
+  flexDirection: "column",
+  gap: Theme.Spacing.medium,
+});
+
+const TemporaryEffectsTable = styled.ul({
+  display: "table",
+  borderCollapse: "collapse",
+  textAlign: "left",
+  gap: Theme.Spacing.small,
+  td: {
+    padding: Theme.Spacing.small,
+    border: `1px solid ${Theme.Surface[200]}`,
+    "&.item-name": {
+      width: "70%",
+    },
+    "&.item-duration": {
+      width: "30%",
+    },
+  },
+  li: {
+    p: {
+      overflow: "hidden",
+      textOverflow: "ellipsis",
+      whiteSpace: "nowrap",
+    },
+  },
```

Comment:
This style seems unused (I can't spot any `<li>` tags at least)

Reply:
Hah, I changed my mind from ul/li to table and forgot to clean up, oops... 1am coding vibes

Reply:
Exactly what i suspected 😄